lib/init.lua

```diff
@@ -39,6 +40,7 @@ apply(Roact, {
 	PureComponent = PureComponent,
 	Event = Event,
 	Change = Change,
+	Instrumentation = Instrumentation,
```

Comment:
We can probably only expose `printStats` here instead of exposing the entire module.